src/components/game/index.jsx

```diff
@@ -77,15 +78,19 @@ class GameScreen extends React.Component {
     this.refs.controls.update();
     this.refs.stats.forceUpdate();
 
-    const cameraRotated = this.prevCameraRotation === null ||
-      !this.prevCameraRotation.equals(this.camera.quaternion);
+    const cameraMoved =
+      this.prevCameraRotation === null ||
+      this.prevCameraPosition === null ||
+      !this.prevCameraRotation.equals(this.camera.quaternion) ||
+      !this.prevCameraPosition.equals(this.camera.position);
 
-    session.world.animate(this.clock.getDelta(), this.camera, cameraRotated);
+    session.world.animate(this.clock.getDelta(), this.camera, cameraMoved);
```

Comment:
Let's keep it as it is for now :+1: Perhaps it's even possible to mark it on the camera itself – on `userData` perhaps? That would reduce the amount of stuff we're passing around.